realm/kotlin-extensions/lib/package-list.txt

```diff
@@ -0,0 +1,229 @@
+$dokka.format:kotlin-website-html
```

Comment:
What is this file?

Reply:
It was a workaround for https://github.com/Kotlin/dokka/issues/709, but it has been fixed. It can be removed again. There is a TODO in the Dokka task in the gradle file